### PR TITLE
feat(users,dbs): Allow to configure users and DBs

### PR DESCRIPTION
**Pull request by haslersn**

### CODE REVIEW

File: tasks/install.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@
       - postgresql
       - postgresql-client
       - postgresql-doc
+      - libpq-dev
```
Comment (neumantm) [inline tasks/install.yml:8]: Why is this needed?

**Review by neumantm — APPROVED** [review]

File: tasks/install.yml (hunk 1 above)
Comment (haslersn) [reply tasks/install.yml:8]: I got an error when running synapse_port_db without this.
